src/test/python/XMLsundry.py

```diff
@@ -0,0 +1,14 @@
+from xml.etree import ElementTree as ET
+from src.main.python.pythonparser.pythonparser_3 import *
```

Comment:
1. Add copyright, please

2. Add an empty row between imports because they are from the different groups: an internal library and an internal module.

Reply:
Sorry, this file ( XMLSundry.py) is unnecessary and does not belong to the project. I didn’t have to commit it, so I’m deleting it.